src/winter_onboarding_2021/fleet_management_service/handlers/fleet.clj

```diff
@@ -0,0 +1,32 @@
+(ns winter-onboarding-2021.fleet-management-service.handlers.fleet
+  (:require [winter-onboarding-2021.fleet-management-service.models.fleet :as fleet-model]
+            [clojure.spec.alpha :as s]
+            [winter-onboarding-2021.fleet-management-service.specs :as specs]
+            [ring.util.response :as response]
+            [winter-onboarding-2021.fleet-management-service.views.fleet :as views]))
+
+(def success-flash
+  {:flash {:success true
+           :style-class "alert alert-success"
+           :message "Fleet created successfully!"}})
+(def error-flash
+  {:flash {:error true
+           :style-class "alert alert-danger"
+           :message "Could not create fleet, try again!"}})
+
+(defn create-fleet [req]
```

Comment:
I think we've called it request in most other places, can we stick to that?

Also, somewhat minor since we still have to do a lookup in this instance, but we can destructure `user` and `form-params`

Reply:
I guess req is fine. Most places we have use req instead of request. 
I am not clear about the suggestion in the second line but. Can u please clarify @neena?